azext_iot/_params.py

```diff
@@ -70,6 +76,8 @@ def load_arguments(self, _):
     with self.argument_context('iot hub') as context:
         context.argument('target_json', options_list=['--json', '-j'],
                          help='Json to replace existing twin with. Provide file path or raw json.')
+        context.argument('policy_name', options_list=['--policy-name', '-po'],
```

Comment:
I think the [original releases](https://github.com/Azure/azure-iot-cli-extension/blob/master/azext_iot/_params.py#L44) use -po probably because of a last minute name change (like from --policy to --policy-name). I don't have a lot of opinion on this though and can change to -pn to fit our convention.

Reply:
Thoughts?

Reply:
i am thinking -pn is better so users would feel it being more consistent